### ISSUE TITLE
Trivial program does not learning during training after refresh

### ISSUE THREAD

**Issue by ToonTalk** [tensorflow/tfjs#918]
#### TensorFlow.js version
0.13.3
#### Browser version
Version 70.0.3538.102 (Official Build) (64-bit)
#### Describe the problem or feature request
The following sometimes works when refreshing the page and sometimes there is no learning (loss stays at 21 from epoch 0).
It differs from https://js.tensorflow.org/#getting-started only in the addition of activation: 'relu' and the callbacks to fit to watch the progress. activation: 'relu' may be critical here (and yes it isn't needed in this example).
Problem persists if backend set to cpu. Also with latest version of FireFox.
#### Code to reproduce the bug / link to feature request
<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs"> </script>
    <script>
      const model = tf.sequential();
      model.add(tf.layers.dense({units: 1, inputShape: [1], activation: 'relu'}));
      model.compile({loss: 'meanSquaredError', optimizer: 'sgd'});
      const xs = tf.tensor2d([1, 2, 3, 4], [4, 1]);
      const ys = tf.tensor2d([1, 3, 5, 7], [4, 1]);
      let callbacks = {onEpochEnd: async (epoch, h) => {
          console.log(epoch, h);
      }};
      model.fit(xs, ys, {epochs: 10,
                callbacks: callbacks}).then(() => {
        model.predict(tf.tensor2d([5], [1, 1])).print();
      });
    </script>

**Comment by ToonTalk** [comment]
Thanks. You are right I can't reproduce it with tanh or sigmoid. I see now the random initialisation is why it behaves differently.

tf.layers.dense only takes a string argument so if I wanted leakyRelu would I have to stop using tf.layers.dense and use lower level operations?

**Comment by caisq** [comment]
I'm pretty sure that 'leakyrelu' is not supported as a string activation even in Python Keras. It's accessible via tf.layers.* though.

**Comment by davidsoergel** [comment]
See also https://github.com/tensorflow/tfjs/issues/1093
